server/app/models.py

```diff
@@ -1246,11 +1274,13 @@ def get_scores(self):
         else:
             members = [self.submitter]
         for member in members:
-            email = member.get().email[0]
+            mem_email = member.get().email[0]
+            grader_email = None
+            
             for score in self.submission.get().score:
                 all_scores.append([email,
                         score.score,
                         score.message,
-                        score.grader.get().email[0],
+                        grader_email,
                         score.tag])
```

Comment:
@alvinwan  I added this because Prof. DeNero said we couldn't depend on autograded scores having a grader attribute. Now it seems like master has been changed so that also autograded scores have a grader. So my changes aren't necessary now, but make the code able to handle scores without grader if those might exist in the future.